### ISSUE TITLE
YouTube is unstable with AdGuard enabled

### ISSUE THREAD

**Issue by Sirix1995** [AdguardTeam/AdGuardForSafari#884]
### Please answer the following questions for yourself before submitting an issue.

- [x] I am running the latest version
- [x] I made sure all content blockers are enabled in Safari settings
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### AdGuard version

1.11.14

### Environment

```markdown
- OS:Mojave 10.14.6
- Safari version:14.1.2
```

### What filters do you have enabled?

AdGuard Base filter, AdGuard French filter

### Issue Details

Steps to reproduce:
1. Load a Youtube page
2. Navigate threw a link (search, video...)

### Expected Behavior

Navigating and load videos without the need of deactivate totally AdGuard.

### Actual Behavior

If I change page on YouTube, if AdGuard is active, YouTube stops and says I don't have any Internet connexion. But when checking on my network states, all is working fine. If I load a video page, the loading will acts strangely (it looks corrupted), and then, nothing happens. If I shut down AdGuard totally, YouTubes behaves perfectly normally.

### Screenshots

<details><summary>Screenshot 1: stuck on no connection page</summary>

(https://cdn.discordapp.com/attachments/605532970081845347/1147950454836297858/Capture_decran_2023-09-03_a_19.41.09.png)

</details>

<details><summary>Screenshot 2: stuck video loading</summary>

(https://cdn.discordapp.com/attachments/605532970081845347/1147950455494807605/Capture_decran_2023-09-03_a_19.41.34.png)

</details>

**Comment by aberkvam** [comment]
Agreed.  AdGuard 1.11.14 in Safari 14.1.2 on macOS 10.14.6 makes YouTube unusable.  Videos either refuse to play or only play the first few seconds.  Disabling AdGuard on youtube.com just leads to other errors.  Completely disabling AdGuard was the only way I could get YouTube to work.

**Comment by crgmrgn** [comment]
For reference, behaviour persists after upgrade to v1.11.15 standalone release (on macOS 10.13.6), w. Base Filter v.2.3.3.86 dated 06/10/2023 ...

**Comment by vitamin-c-gmbh** [comment]
Same problem experienced under 14.0 (23A344) and Safari 17.0 (19616.1.27.211.1). Had to reboot to get things back up and running.

**Comment by crgmrgn** [comment]
Latest beta release (v1.1.6beta) seems to resolve the YT "offline" on following a second link issue here, so nice work. 

Environ is macOS 10.13.6, Safari 13.1.2 and base filter v2.3.9.17 (11/11/2023).

**Comment by crgmrgn** [comment]
Surely this is addressed by v1.1.6 release and can be closed now?

**Comment by windwak3r** [comment]
AdGuard Mini is now available in the App Store! We'll close the issue for now. If the problem continues to persist on AdGuard Mini, please leave a comment.
